src/sql/parts/objectExplorer/common/objectExplorerService.ts

```diff
@@ -683,6 +699,9 @@ export class ObjectExplorerService implements IObjectExplorerService {
 
 	private async setNodeExpandedState(treeNode: TreeNode, expandedState: TreeItemCollapsibleState): Promise<void> {
 		treeNode = await this.getUpdatedTreeNode(treeNode);
+		if (!treeNode) {
+			return Promise.resolve();
```

Comment:
The node not existing is an error case for this method and the ones below. I'd think we would want them to throw an error. Why is this better?

Reply:
This code tries to find the node in the existing servers view. So for this new tree, it definitely won't exist, which was a problem I couldn't code around without modifying the oe service.

Reply:
The proper solution is probably to refactor the OE service to separate it from the ServerTreeView, but that would be a much larger change.